pkg/operators/clustermanager/options.go

```diff
@@ -31,9 +35,27 @@ func (o *Options) RunClusterManagerOperator(ctx context.Context, controllerConte
 	// kubeInformer is for 3 usages: configmapInformer, secretInformer, deploynmentInformer
 	// After we introduced hosted mode, the hub components could be installed in a customized namespace.(Before that, it only inform from "open-cluster-management-hub" namespace)
 	// It requires us to add filter for each Informer respectively.
-	// TODO: Wathc all namespace may cause performance issue.
+	// TODO: Watch all namespace may cause performance issue.
 	kubeInformer := informers.NewSharedInformerFactoryWithOptions(kubeClient, 5*time.Minute)
```

Comment:
why do we still need this?

Reply:
statusController and clusterManagerController use it to watch deployments